build/config/compiler/BUILD.gn

```diff
@@ -309,6 +310,26 @@ config("compiler") {
     ]
   }
 
+  if (is_wasm) {
```

Comment:
I think the pattern here is to put these common flags in the toolchain definition.

Reply:
Is there a good example of that?  Maybe I'm misunderstanding but I'm not seeing it in other toolchain definitions.